doc/classes/ScrollContainer.xml

```diff
@@ -102,6 +102,9 @@
 		<constant name="SCROLL_MODE_SHOW_NEVER" value="3" enum="ScrollMode">
 			Scrolling enabled, scrollbar will be hidden.
 		</constant>
+		<constant name="SCROLL_MODE_RESERVE" value="4" enum="ScrollMode">
+			Combines [constant SCROLL_MODE_AUTO] and [constant SCROLL_MODE_SHOW_ALWAYS]. The scrollbar is only visible if necessary, but the content size is adjusted as if it was always visible.
```

Comment:
I feel like this description doesn't make the purpose and function clear enough, would be better to emphasize that it results in the content not being resized